src/Presentation/Nop.Web/Views/Catalog/GetProductOverview.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@{
+    Layout = null;
+}
+
+@model ProductOverview
+
+<div id="float-container-product-overview">
+    <div class="float-child-product-overview">
+        <h3>Title: </h3>
```

Comment:
redo this please
https://bulma.io/documentation/form/general/ have a look here for example how is done a proper markup

`<div class="field">
  <label class="label">Name</label>
  <div class="control">
    <input class="input" type="text" placeholder="Text input">
  </div>
</div>`
each field is separated and individual one